Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -148,7 +163,6 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
      * @param motor used to access the motor to set and give data
      */
     private void updateMotorConfig(XCANTalon motor) {
-        motor.configEncoderCodesPerRev((int)encoderCodesProperty.get());
```

Comment:
We should check the docs to see if this is a property that is maintained in flash memory. If it is, we need to make sure that we call this everywhere.